ibm/service/power/resource_ibm_pi_image_export.go

```diff
@@ -31,46 +29,45 @@ func ResourceIBMPIImageExport() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			//required attributes
-			helpers.PICloudInstanceId: {
-				Type:        schema.TypeString,
-				Required:    true,
+			//Arguments
+			Arg_CloudInstanceID: {
 				Description: "PI cloud instance ID",
 				ForceNew:    true,
-			},
-			helpers.PIImageId: {
-				Type:             schema.TypeString,
-				Required:         true,
-				Description:      "Instance image id",
-				DiffSuppressFunc: flex.ApplyOnce,
-				ForceNew:         true,
-			},
-			helpers.PIImageBucketName: {
-				Type:        schema.TypeString,
 				Required:    true,
-				Description: "Cloud Object Storage bucket name; bucket-name[/optional/folder]",
-				ForceNew:    true,
-			},
-			helpers.PIImageAccessKey: {
 				Type:        schema.TypeString,
-				Required:    true,
+			},
+			Arg_ImageAccessKey: {
 				Description: "Cloud Object Storage access key; required for buckets with private access",
-				Sensitive:   true,
 				ForceNew:    true,
-			},
-
-			helpers.PIImageSecretKey: {
-				Type:        schema.TypeString,
 				Required:    true,
-				Description: "Cloud Object Storage secret key; required for buckets with private access",
 				Sensitive:   true,
-				ForceNew:    true,
+				Type:        schema.TypeString,
 			},
-			helpers.PIImageBucketRegion: {
+			Arg_ImageBucketName: {
+				Description: "Cloud Object Storage bucket name; bucket-name[/optional/folder]",
+				ForceNew:    true,
 				Type:        schema.TypeString,
+				Required:    true,
+			},
+			Arg_ImageBucketRegion: {
 				Description: "Cloud Object Storage region",
 				ForceNew:    true,
 				Required:    true,
+				Type:        schema.TypeString,
+			},
+			Attr_ImageID: {
```

Comment:
This is the wrong constant. Before and after the refactor the value of the constants should be the same. The only thing that should change is the name. The constant here should be called Arg_ImageID with value "pi_image_id"

Reply:
Added